_opensearch/rest-api/snapshots/create-repository.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Create Snapshot Repository
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+## Create snapshot repository
+
+Registers a new repository to store snapshots. Repositories comprise two types:
+
+* File system (`fs`)
+
+* Amazon S3 bucket (`s3`)
+
+To learn more about repositories, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
```

Comment:
Is registering a repository equivalent to creating it in this API? We use two terms: create and register, but I think it's the same API operation. If so, we should change the wording to be uniform here and in "Take snapshots".

Reply:
Thank you! I changed terminology in create-repository.md to say "register" instead of "create." Text in create-snapshot.md did not have to be changed.